src/test/scala/mesosphere/marathon/integration/NetworkPartitionIntegrationTest.scala

```diff
@@ -2,7 +2,7 @@ package mesosphere.marathon.integration
 
 import mesosphere.marathon.integration.facades.ITEnrichedTask
 import mesosphere.marathon.integration.setup._
-import org.scalatest.{BeforeAndAfter, GivenWhenThen, Matchers}
+import org.scalatest.{ BeforeAndAfter, GivenWhenThen, Matchers }
```

Comment:
This was automatically done by sbt when running our tests, because the current state of the repo doesn't follow the style rules.

https://github.com/mesosphere/marathon/pull/4068 has the same change. I'll merge that PR.